src/modules/pivotTable/useParentSize.js

```diff
@@ -20,13 +21,18 @@ export const useParentSize = (
                 height: el.clientHeight,
             })
         }
+
         onResize(el)
 
+        if (renderId) {
+            setSize({ width: 0 })
```

Comment:
Could we set width to undefined instead?

Reply:
Sure. Tested and it works fine.